src/routes/+page.svelte

```diff
@@ -53,21 +57,7 @@
       href="{base}/resources"
       class="block w-fit p-4 rounded-lg border-2 border-green-500 dark:border-green-700 text-green-700 dark:text-green-500 hover:text-black hover:bg-green-500 dark:hover:text-white dark:hover:bg-green-900 transition-colors"
     >
-      <svg
-        xmlns="http://www.w3.org/2000/svg"
-        fill="none"
-        viewBox="0 0 24 24"
-        stroke-width="1.5"
-        stroke="currentColor"
-        class="w-6 h-6 inline"
-      >
-        <path
-          stroke-linecap="round"
-          stroke-linejoin="round"
-          d="M12 6.042A8.967 8.967 0 006 3.75c-1.052 0-2.062.18-3 .512v14.25A8.987 8.987 0 016 18c2.305 0 4.408.867 6 2.292m0-14.25a8.966 8.966 0 016-2.292c1.052 0 2.062.18 3 .512v14.25A8.987 8.987 0 0018 18a8.967 8.967 0 00-6 2.292m0-14.25v14.25"
-        />
-      </svg>
-
+      <Book class="w-6 h-6 inline" />
```

Comment:
the positioning on this one is odd when comparing, the icon and text touching vs the youtube one 💭 
<img width="880" alt="Screen Shot 2023-07-26 at 7 54 14 PM" src="https://github.com/ClimateTown/knowledge-hub/assets/4835191/b733d45f-7719-425d-b28a-b6013b504b3c">

these could use the buttonLinks component also 
```
 <ButtonLinks link={true} url="{base}/resources" version="hollow" color="green" extraClasses="block w-fit !p-4 !gap-3">
      <Book slot="icon" class="w-6 h-6 inline" />
      <span slot="label">Resources</span>
    </ButtonLinks>
<ButtonLinks link={true} url="{base}/youtube" version="hollow" color="red" extraClasses="block w-fit !p-4">
      <Play slot="icon" class="w-6 h-6 inline" />
      <span slot="label">YouTube Feed</span>
    </ButtonLinks>
```
after:
<img width="1044" alt="Screen Shot 2023-07-26 at 8 04 16 PM" src="https://github.com/ClimateTown/knowledge-hub/assets/4835191/3002fedf-f623-41e7-a0a7-df7b4bad4282">

(screenshot above also has `flex flex-col` on the card wrapper and `grow` on the div after the heading to force the buttons to align to the bottom of the card)

Reply:
looks great! Much cleaner with the `ButtonLinks` component